app/assets/javascripts/controllers/TagCtrl.coffee

```diff
@@ -0,0 +1,26 @@
+@bruseApp.controller 'TagCtrl', ['$scope', '$rootScope', '$location', '$routeParams', 'TagHandler', ($scope, $rootScope, $location, $routeParams, TagHandler) ->
+  if !$rootScope.new_files || !$rootScope.new_files.length
+    $location.path('/service/'+$routeParams.identity_id+'/files/add')
+
+  $scope.files = $rootScope.new_files
+  $scope.tags = ""
+
+  $scope.save = ()->
+    success = true
+    tagsForAll = _.compact $scope.tags.split ' '
+    ###
+    For each file, split the input field by space into an array.
+    Remove empty strings with the _.compact method.
+    And lastly combine two arrays with concat.
+    ###
+    $scope.files.forEach((file)->
```

Comment:
(Minor bullshit comment number two: [Spaces before function arrows!](Always include a space before and after a function arrow.))